### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by monicalopezgris**

### CODE REVIEW

File: documentation/devon4ng-adding-custom-functionality.asciidoc [hunk 1]
```diff
@@ -948,11 +958,12 @@ export class LoginService {
 
         pageable.pageNumber = 0;
         pageable.pageSize = 1;
+        pageable.sort= [];
```
Comment (sjimenez77) [inline documentation/devon4ng-adding-custom-functionality.asciidoc:961]: There is no space between `pageable.sort=`

File: documentation/devon4ng-adding-custom-functionality.asciidoc [hunk 2]
```diff
@@ -1189,6 +1200,7 @@ export class AccessCode {
     endTime?: string;
     visitorId: number;
     queueId: number;
+    content: any;
```
Comment (sjimenez77) [inline documentation/devon4ng-adding-custom-functionality.asciidoc:1203]: Why not do we create a type in order not to use the `any` type? In general it is a bad practice.

File: documentation/devon4ng-adding-custom-functionality.asciidoc [hunk 3]
```diff
@@ -1200,6 +1212,7 @@ export class Queue {
     minAttentionTime: string;
     active: boolean;
     customers: number;
+    content: any;
```
Comment (sjimenez77) [inline documentation/devon4ng-adding-custom-functionality.asciidoc:1215]: Same here regarding `any` type.

**Review by sjimenez77 — CHANGES_REQUESTED** [review]
Hi @monicalopezgris, I have posted a few comments to improve this PR.

File: angular/src/app/shared/backendModels/interfaces.ts [hunk 4]
```diff
@@ -1,61 +1,80 @@
-export class FilterVisitor {
-    pageable: Pageable;
-    username: string;
-    password?: string;
-}
-export class FilterAccessCode {
-    pageable: Pageable;
-    visitorId: Number;
+export class Visitor {
+  [x: string]: any;
```
Comment (mperezor19) [inline angular/src/app/shared/backendModels/interfaces.ts:2]: Is not possible to create an specific type?

**Review by mperezor19 — COMMENTED** [review]
Check comment

**Comment by monicalopezgris** [comment]
Created new request

**Pull request merged**